server/datastore/mysql/operating_system_vulnerabilities.go

```diff
@@ -50,13 +50,14 @@ func (ds *Datastore) ListVulnsByOsNameAndVersion(ctx context.Context, name, vers
 				MIN(software_cve.created_at) created_at
 			FROM
 				software_cve
-				JOIN software ON software.id = software_cve.software_id
-				JOIN software_titles ON software_titles.id = software.title_id
-				JOIN host_software ON host_software.software_id = software.id
-				JOIN host_operating_system ON host_operating_system.host_id = host_software.host_id
-				JOIN operating_systems ON operating_systems.id = host_operating_system.os_id
+				-- JOIN software ON software.id = software_cve.software_id
+				-- JOIN software_titles ON software_titles.id = software.title_id
+				-- JOIN host_software ON host_software.software_id = software.id
+				-- JOIN host_operating_system ON host_operating_system.host_id = host_software.host_id
```

Comment:
Drop the comments out since they may drift from how we're actually running this.

Also, how do we determine host counts?